pkgs/development/libraries/miniz/default.nix

```diff
@@ -1,29 +1,40 @@
-{ lib, stdenv, fetchFromGitHub, cmake }:
+{ lib
+, fetchFromGitHub
+, nix-update-script
+, stdenv
+, testers
+, validatePkgConfig
+, cmake
+}:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "miniz";
-  version = "2.2.0";
+  version = "3.0.2";
 
   src = fetchFromGitHub {
     owner = "richgel999";
-    repo = pname;
-    rev = version;
-    sha256 = "sha256-7hc/yNJh4sD5zGQLeHjowbUtV/1mUDQre1tp9yKMSSY=";
+    repo = finalAttrs.pname;
+    rev = finalAttrs.version;
+    sha256 = "sha256-3J0bkr2Yk+MJXilUqOCHsWzuykySv5B1nepmucvA4hg=";
   };
+  passthru.updateScript = nix-update-script {};
 
-  nativeBuildInputs = [ cmake ];
+  nativeBuildInputs = [ cmake validatePkgConfig ];
 
```

Comment:
```suggestion

  strictDeps = true;

```

Reply:
https://nixos.org/manual/nixpkgs/stable/#ssec-stdenv-dependencies-reference

Reply:
The reference was removed, I still don't know why. I will search the commit logs.

Summarizing a bit, `strictDeps =  true` enforces a clearer separation between the various `*buildInputs`. It is helpful e.g. in cross-compilation and porting to other environments.

```
 By default, the run-time/build-time distinction is just a hint for mental clarity, but with strictDeps set it is mostly enforced even in the native case.
```